.github/actions/make-in-docker/action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
     required: true
 runs:
   using: docker
-  image: docker://webplatformtests/wpt.fyi:latest
+  image: docker://webplatformtests/wpt.fyi:go114
```

Comment:
This will change back to `latest` after a weekly update in [docker-update.yml](https://github.com/web-platform-tests/wpt.fyi/blob/master/.github/workflows/docker-update.yml)?

Reply:
No, I'll have to revert it myself :)